astro/src/components/get-started/GitPodStart.astro

```diff
@@ -15,15 +15,18 @@ export interface Props {
 const { kickstartUri, api = false } = Astro.props as Props;
 ---
 <dd>
+
+<Aside type="note" title="GitPod Registration">  Note that the first time you start a GitPod workspace you will need to answer some demographic questions, but after that it will run without interruption.</Aside>
+
+<p>Click on the GitPod button below to start up your environment.  It will take a few minutes to bring up the servers, so please be patient until the login page loads into the environment.</p>
```

Comment:
```suggestion
<p>Click on the GitPod button below to start up your environment. It will take a few minutes to bring up the servers, so please be patient until the login page loads into the environment.</p>
```